app.js

```diff
@@ -1,7 +1,37 @@
 const express = require("express")
 const { getApi } = require("./controllers/app.controller")
+const { getTopics } = require("./controllers/topics.controller")
 const app = express()
 
+app.use((req, res, next) => 
+    next()
+)
+
 app.get("/api", getApi)
 
+app.get("/api/topics", getTopics)
+
+
+
+app.use('/*', (req, res) => {
+    res.status(404).send({ msg: 'Route not found' })
```

Comment:
Nice one here to pass your test 🥇 Is there a way we could we neaten up our app file potentially?